### ISSUE TITLE
Duplicate keys is DELL device types

### ISSUE THREAD

**Issue by jensrudolf** [netbox-community/devicetype-library#311]
The following device type definitions contain duplicate keys, which are illegal as of [YAML spec 1.2](https://yaml.org/spec/1.2/spec.html#id2764044)

```
device-types/Dell/PowerEdge_R640.yaml:18:1: [error] duplication of key "console-server-ports" in mapping (key-duplicates)
device-types/Dell/PowerEdge_R620.yaml:18:1: [error] duplication of key "console-server-ports" in mapping (key-duplicates)
device-types/Dell/PowerEdge_R630.yaml:18:1: [error] duplication of key "console-server-ports" in mapping (key-duplicates)
device-types/Dell/PowerEdge_R720.yaml:18:1: [error] duplication of key "console-server-ports" in mapping (key-duplicates)
device-types/Dell/PowerEdge_R740.yaml:18:1: [error] duplication of key "console-server-ports" in mapping (key-duplicates)
device-types/Dell/PowerEdge_R730.yaml:18:1: [error] duplication of key "console-server-ports" in mapping (key-duplicates)
```

I would be willing to provide a pull request if a fix is appreciated.

**Comment by kbreit** [comment]
@jensrudolf I am looking at this one and not seeing duplicate keys in a few of these files. Are you seeing them in the latest versions still?

**Comment by jensrudolf** [comment]
This issue has been fixed by #317 and #319.
